src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2201,6 +2202,11 @@ public function getBinaryStream($path)
             array($nodeId, $propertyName, $this->workspaceName)
         );
 
+        if (count($data) === 0) {
+            throw new RepositoryException('No binary data found in stream');
```

Comment:
what exactly does that mean? and what if i created a binary multivalue property and set it to an empty array? it seems that this is allowed...

or is this meant as sanity check for database inconsistency?